.github/workflows/libclang-python-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Libclang Python Binding Tests
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'clang/bindings/python/**'
+      - 'clang/tools/libclang/**'
+      - 'clang/CMakeList.txt'
+      - 'cmake/**'
```

Comment:
Why rebuild when the cmake infrastructure changes? Changes there are quite rare and I don't think are likely to impact the libclang python bindings.

Reply:
Anyways. I dropped `cmake/*` from the paths list. Arguably, there are many other cmake include locations that also weren't covered (like clang/cmake/* and llvm/cmake/*). This was starting to feel like whac-a-mole.

Reply:
Yep. I've had similar experiences working on CI for other parts of the project. If it's a frequent occurrence that CMake changes break the libclang builds (or even if it just happens more than once), we can revisit this point.

Reply:
That's fair 👌